crates/mfem-sys/build.rs

```diff
@@ -62,12 +63,12 @@ impl MfemConfig {
         // Pre-installed MFEM will be checked for compatibility using semver rules.
         let version_req = semver::VersionReq::parse(">=4.6").expect("Invalid version constraint");
 
-        println!("cargo:rerun-if-env-changed=DEP_MFEM_ROOT");
+        println!("cargo:rerun-if-env-changed=MFEM_DIR");
```

Comment:
question: What's the reason for changing the name of this env var?

Reply:
It was because I thought it was the expected name for people setting it outside the crate: see e.g. https://mfem.org/building/   If you do not like it, I can revert it (in the autocxx branch).